samples/get_portgroup.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Written by Chris Arceneaux
+# GitHub: https://github.com/carceneaux
+# Email: carcenea@gmail.com
+# Website: http://arsano.ninja
+#
+# Note: Example code For testing purposes only
+#
+# This code has been released under the terms of the Apache-2.0 license
+# http://opensource.org/licenses/Apache-2.0
+
+"""
+Python program for retrieving a port group for both VSS and DVS
+"""
+
+import atexit
+
+from tools import cli
+from pyVim import connect
+from pyVmomi import vmodl
+from pyVmomi import vim
+
+
+def get_args():
+    """
+    Adds additional args for retrieving a port group
+
+    -pg portgroupname
+    """
+    parser = cli.build_arg_parser()
+
+    # because -p is reserved for 'password'
+    parser.add_argument('-pg', '--portgroupname',
```

Comment:
Hi @carceneaux , Actually my comment was, lets not use -pg, we can just keep --portgroupname

Reply:
I guess I don't understand. My current method allows for both `-pg` and `--portgroupname` to be used. Isn't that better as it allows someone a shorter alternative if they prefer? All scripts I've contributed to this project thus far have followed the same format of _short_ and _long_ arguments.

Reply:
Usually what I prefer is, have short command line arg only if there are no common char another argument.
Ex: -p is already there. if someone adds -g argument to your sample it will conflict. So I said we can have only long one.
Anyway it is fine, not a big factor to consider. I will merge this sample.

Thanks for contributing